cmd/ctr/commands/images/pull.go

```diff
@@ -67,6 +67,10 @@ command. As part of this process, we do the following:
 			Usage:  "(Deprecated: use skip-metadata) Pull metadata for all platforms",
 			Hidden: true,
 		},
+		&cli.BoolFlag{
+			Name:  "metadata-only",
+			Usage: "Pull all metadata including manifests and configs",
```

Comment:
The usage doesn't mention what's being excluded here.
```suggestion
			Usage: "Pull all metadata including manifests and configs, without image layers",
```

I might be wrong about what's being excluded here, which is why I suggest it could be clearer. I see that `ctr fetch --metadata-only` already has this usage description, so maybe this has already been discussed?